coding-concepts.js

```diff
@@ -11,34 +11,35 @@
 const cohort = "Foxtrot 2023"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: it will turn it into an array with each letter separated by quotes
+
+// b) Verify and explain: that happened, it is because split is a built in method to turn strings into arrays, and we didn't put a space in there to separate the word and numbers, so we just got each letter separated by quotes. 
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
   `Hello, ${name}!`
 }
-// console.log(greeter("LEARN Student"))
+console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: undefined, there is not a variable set that is named LEARN Student
+// b) Verify and explain: exactly that happened, we can't invoke a function to insert a string, we have to set a variable to the function so that it knows what to invoke. 
```

Comment:
It is undefined because there is no `return` in the function. The string does not need to be stored in a variable, as it is just a standalone data type that is being used as an argument.